### PR TITLE
refactor: remove test env

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of the Leather — download [extension builds](https://github.com/leather-wallet/extension/actions/runs/6155993196).

See title

### CODE REVIEW

File: scripts/generate-manifest.js [hunk 1]
```diff
@@ -29,12 +32,14 @@ const environmentIcons = {
 };
 
 const contentSecurityPolicyEnvironment = {
+  testing: `default-src 'none'; connect-src *; style-src 'unsafe-inline'; img-src 'self' data: https:; script-src 'self' 'wasm-unsafe-eval'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';`,
   development:
     "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'; frame-src 'none'; frame-ancestors 'none';",
   production: `default-src 'none'; connect-src *; style-src 'unsafe-inline'; img-src 'self' data: https:; script-src 'self' 'wasm-unsafe-eval'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';`,
 };
```
Comment (pete-watters) [inline scripts/generate-manifest.js:40]: I'm curious, what is this doing? 🤔  

Do we need to change icons for tests? Can we add `-preview` here in `defaultIconEnvironment` or do we need to do it in `environmentIcons`

**Review by pete-watters — APPROVED** [review]

File: scripts/generate-manifest.js (hunk 1 above)
Comment (alter-eggo) [reply scripts/generate-manifest.js:40]: I just copy pasted it from production, as I believed it need to be same as in production env 🤔

File: scripts/generate-manifest.js [hunk 2]
```diff
@@ -29,12 +32,14 @@ const environmentIcons = {
...
 
 const defaultIconEnvironment = {
+  testing: 'assets/connect-logo/Stacks128w.png',
```
Comment (fbwoolf) [inline scripts/generate-manifest.js:42]: It seems like we should update all these icons to Leather?

Comment (alter-eggo) [reply scripts/generate-manifest.js:42]: better to do it in separate pr

**Review by fbwoolf — APPROVED** [review]

**Pull request merged**